src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -1188,6 +1209,10 @@ impl<'a> Linker for PtxLinker<'a> {
         self.cmd.arg("--debug");
     }
 
+    fn strip(&mut self) {
+        // noop
```

Comment:
Instead of `noop`, I would mention here that the PTX Linker doesn't support stripping on linking.

CC: @denzp, is this true?